specification/api-metrics-user.md

```diff
@@ -9,9 +9,23 @@ TODO: Add a table of contents.
 
 ## Overview
 
-Metric instruments are the entry point for application and framework
-developers to instrument their code using counters, gauges, and
-measures.
+Metric instruments are the entry point for application and framework developers to instrument their code using counters, gauges, and measures.
+Metrics are created by calling methods on a `Meter` which is in turn created by a global `MeterFactory`.
```

Comment:
Didn't we change to a "Registry" term in the tracing world?

Reply:
we're updating the otep before the spec